src/Agents.jl

```diff
@@ -17,7 +17,8 @@ include("spaces/nothing.jl")
 include("spaces/graph.jl")
 include("spaces/grid.jl")
 include("spaces/discrete.jl")
-include("spaces/continuous_space.jl")
+# include("spaces/continuous_space.jl")
```

Comment:
Shouldn't we keep this running? How else will we benchmark this new space with the old? Please keep everything related to the old space. At the end of this PR you can do a massive rename CompartmentSpace -> ContinuousSpace and delete old files if need be, but for now you should set up a benchmark that creates the same model (e.g. SIR billiards) and benchmarks the same functions like move_agent, add_agent, nearby_agents, similarly with how I did it for the forest fire in my PR. Example: https://github.com/JuliaDynamics/Agents.jl/pull/311/commits/b5ea865ee5eea0f03ef9d1e2c3be29fd90e73bfd

Reply:
I'm working on PkgBenchmark integration a little more at the moment. Will try to incorporate something regarding this.